src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Positive;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @NotNull
```

Comment:
I think it`s redundand annotation, because Positive included not null
![image](https://github.com/mate-academy/jv-spring-security/assets/92263491/9f104f8a-ea89-40ee-880c-2cd64ac187fc)